common/modules/farm/events/src/events.rs

```diff
@@ -244,7 +248,7 @@ pub trait EventsModule {
         #[indexed] caller: &ManagedAddress,
         #[indexed] epoch: u64,
         #[indexed] block: u64,
-        #[indexed] timestamp: u64,
+        #[indexed] timestamp: TimestampSeconds,
```

Comment:
Don't know where the type `TimestampSeconds` is defined, but just rename it to `Timestamp`. It's already in seconds wherever else it's used.